modules/action-bar/react/spec/__snapshots__/ActionBar.snapshot.tsx.snap

```diff
@@ -164,9 +164,11 @@ exports[`Action Bar Snapshots renders a action bar with one button 1`] = `
   >
     <button
       className="emotion-2 emotion-3"
+      size="large"
```

Comment:
I wonder if this was an intended change. `size` isn't a valid attribute for `button`

Reply:
We have this issue with some of our other components too (e.g. `Avatar`). I'm not aware of a way to omit props we want for styling from the resulting dom attributes that emotion generates. Do you have any ideas?

Reply:
nvm, found it 🙂 https://emotion.sh/docs/styled#customizing-prop-forwarding